todoman/formatters.py

```diff
@@ -82,7 +82,6 @@ def detailed(self, todo):
             extra_rows += self._columnize('Description', todo.description)
         if todo.location:
             extra_rows += self._columnize('Location', todo.location)
-
```

Comment:
Now I don't understand where is a mistake? and when I check it locally it worked as we want. but why some test cases are failed I really don't understand.